authorization_services/topics/hello-world-create-realm.adoc

```diff
@@ -14,13 +14,24 @@ image:{project_images}/getting-started/hello-world/create-realm.png[alt="Realm h
 +
 The user list page displays where you can create a user.
 
+ifeval::[{project_community}==true]
+. Click *Create user*.
+endif::[]
+ifeval::[{project_product}==true]
 . Click *Add User*.
+endif::[]
 
 . Complete the *Username*, *Email*, *First Name*, and *Last Name* fields.
 
 . Toggle *User Enabled* to *ON*
 
+ifeval::[{project_community}==true]
+. Click *Create*.
+endif::[]
+ifeval::[{project_product}==true]
 . Click *Save*.
+endif::[]
+
 +
 .Add User
 image:{project_images}/getting-started/hello-world/create-user.png[alt="Add User"]
```

Comment:
Can you slightly increase this screen shot so that "identity providers" in the menu is not cut off?